homeassistant/components/eps/const.py

```diff
@@ -0,0 +1,22 @@
+"""Constants for the iAlarm integration."""
```

Comment:
Stale docstring

Reply:
I agree but it seems that docstrings are compulsory for black :/